### DISCUSSION TITLE
[QUESTION]:ilogtail2.0.0使用damonset的方式启动时报错，ilogtail exited unexpectedly

### DISCUSSION THREAD

**Discussion by onejimmyboy** [alibaba/ilogtail#1344] [Help]
版本：2.0.0
部署方式：daemonsets
错误信息：
ilogtail started. pid: 11
/usr/local/ilogtail/ilogtail_control.sh: line 86:    11 Illegal instruction     (core dumped) ( $bin_file $@ )
ilogtail exited unexpectedly
操作系统：centos7 x86_64

**Comment by messixukejia** [comment]
gdb看下core dump是哪里

**Comment by henryzhx8** [comment]
可以看下在宿主机上直接运行是否正常

**Comment by guochunxing** [comment]
我也碰到了相同的问题，两个 k8s 集群都这样

**Comment by linrunqi08** [comment]
@onejimmyboy @guochunxing  可以在宿主机上执行一下lscpu看下吗

**Reply by XuHandsome** [reply]
[root@k8s-master-v30 ~]# lscpu
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                4
On-line CPU(s) list:   0-3
Thread(s) per core:    1
Core(s) per socket:    4
Socket(s):             1
NUMA node(s):          1
Vendor ID:             AuthenticAMD
CPU family:            15
Model:                 6
Model name:            Common KVM processor
Stepping:              1
CPU MHz:               3193.998
BogoMIPS:              6387.99
Hypervisor vendor:     KVM
Virtualization type:   full
L1d cache:             64K
L1i cache:             64K
L2 cache:              512K
L3 cache:              16384K
NUMA node0 CPU(s):     0-3
Flags:                 fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx lm rep_good nopl cpuid extd_apicid tsc_known_freq pni cx16 x2apic hypervisor cmp_legacy 3dnowprefetch vmmcall

**Reply by XuHandsome** [reply]
同样的问题

**Comment by linrunqi08** [comment]
可以看下这个https://github.com/alibaba/ilogtail/discussions/1398
